HealthyPlanners/Urban-Sherpa
# Refactor suspense

**Task**
Refactor Suspend Component so it's less confusing in implementation.
See Issue #12 

**To Test**
Enter the user section and observer the placeholder. Do a search, select an address, and observe the placeholder. Click "Get Health" and observe the placeholder. It works just as before (better actually) and the whole thing has been rewritten.

**The Files**
**UserSection.jsx**
Removed the bang required by suspend previous implementation. All other changes are linting via Prettier save (I guess it's working now?)

**Suspend.jsx**
Refactored the Logic to render placeholder when condition is true (ie isLoading) and added prop-types to satisfy linting. One linting error remains on the useEffect return callback that I can't figure out.
